Documentation/Design/Classes/Member Variables.md

```diff
@@ -0,0 +1,277 @@
+# Member Variable Retrieval
+
+## Feature Overview
+
+Classes may define member variables that each instance of that class contains.
+Class instances may retrieve those variables.
+
+Many languages, such as C# and Java, guarantee class instances are created with space for each member.
+More dynamic languages may declare members without guaranteeing their existence (TypeScript).
+Others such as Python and Ruby forgo declaring them altogether under certain conditions.
+
+
+## Commands
+
+### `member variable`
+
+`member variable` `:` *`[privacy]`* `instanceName` `variableName`
+
+Retrieving a member variable will be done with the `member variable` command.
+This takes in the variable's privacy optionally, then the instance name and variable name.
```

Comment:
So to confirm, the logic is something like:
if ( 3 parameters )
    1st must match "public"/"protected"/"private"
     2nd is instance name